arm9/src/scrcmd_24.c

```diff
@@ -10,10 +10,6 @@ extern struct UnkStruct_02037CF0* FUN_02037CF0(u32 heap_id, struct UnkSavStruct8
 extern u8 FUN_02037D5C(struct UnkStruct_02037CF0*);
 extern void FUN_02038864(struct UnkSavStruct80*, struct UnkStruct_02088DD8*);
 extern BOOL FUN_0203BC04(struct ScriptContext* ctx);
-extern struct UnkStruct_02088DD8* FUN_02088DD8(u32 heap_id);
```

Comment:
the header file should be included in this file

Reply:
What change are you suggesting? There's already a `#include "unk_02088DD8.h"`.

Reply:
ohh sorry, ignore this then